dodiscover/replearning/gin.py

```diff
@@ -51,7 +52,7 @@ def _extract_edgelists(adj_mat, names):
         return pdag
 
 
-    def fit(self, data, context):
+    def fit(self, data: pd.core.frame.DataFrame, context):
```

Comment:
We do not type hint pandas for now because it is not a stable feature: https://pandas.pydata.org/pandas-docs/version/1.0.0/development/contributing.html#pandas-specific-types